media/js/base/global.js

```diff
@@ -190,3 +190,45 @@ function gaTrack(eventArray, callback) {
         }
     }
 }
+
+
+// Retrieve search params as a object for easier access
+// This is a simple version of https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
+var _SearchParams = function (search) {
```

Comment:
I would like to try and get us out of the habit of adding more and more stuff into `global.js`. I don't think this will be used by that many pages, so I think it could probably be in just it's own file and included in the bundle for `/new`